### PR TITLE
Move oss to open-source

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Moves oss to open-source. This is somewhat temp until we fix the new open source nextjs site.

## What is the current behavior?

oss page

## What is the new behavior?

open-source page

### CODE REVIEW

File: apps/www/lib/redirects.js [hunk 1]
```diff
@@ -1496,6 +1496,11 @@ module.exports = [
     source: '/docs/oss',
     destination: '/oss',
   },
+  {
+    permanent: true,
+    source: '/oss',
+    destination: '/open-source',
+  },
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/lib/redirects.js:1503]: you should keep both redirects.

as the new one will catch any inbound links going to /oss 😄

File: apps/www/pages/open-source/index.tsx [hunk 2]
```diff
@@ -3,12 +3,12 @@ import { useRouter } from 'next/router'
 import { Octokit } from '@octokit/core'
 import DefaultLayout from '~/components/Layouts/Default'
 import SectionContainer from '~/components/Layouts/SectionContainer'
-import maintainers from '../data/maintainers.json'
-import Sponsors from '../components/Sponsors'
+import maintainers from '../../data/maintainers.json'
```
Comment (MildTomato) [inline apps/www/pages/open-source/index.tsx:6]: you can use `~/data/maintainers.json` if you like

**Review by MildTomato — APPROVED** [review]
great stuff. some comments but looks good

**Review by MildTomato — APPROVED** [review]
great stuff. some comments but looks good

**Review by egor-romanov — APPROVED** [review]

**Pull request merged**